lib/reverb/basics.h

```diff
@@ -29,78 +31,16 @@
 	02111-1307, USA or point your web browser to http://www.gnu.org.
 */
 
-#ifndef BASICS_H
-#define BASICS_H
-
-#include <cmath>
-
-#include <stdlib.h>
-#include <string.h>
-
-#include <math.h>
-#include <float.h>
-
-#include <assert.h>
-#include <stdio.h>
-
-#include "util/types.h"
-typedef CSAMPLE sample_t;
-
-// NOTE(rryan): 3/2014 Added these for the MSVC build.
-#include <QtGlobal>
-typedef qint8 int8;
-typedef quint8 uint8;
-typedef qint16 int16;
-typedef quint16 uint16;
-typedef qint32 int32;
-typedef quint32 uint32;
-typedef qint64 int64;
-typedef quint64 uint64;
-
-#define MIN_GAIN 1e-6 /* -120 dB */
-/* smallest non-denormal 32 bit IEEE float is 1.18e-38 */
-#define NOISE_FLOOR 1e-20 /* -400 dB */
-
-/* //////////////////////////////////////////////////////////////////////// */
+#include <cmath> // for M_PI
+#include <cstring> // for memset 
```

Comment:
Why is this in the header if neither `M_PI` nor `memset` are used in it?

Reply:
I have moved them closer to usage.